### ISSUE TITLE
cannot read property SelectorMatcher of undefined tns_modules/nativescript-angular/private_import_compiler.js line 9

### ISSUE THREAD

**Issue by matart15** [NativeScript/nativescript-angular#505]
```
cannot read property SelectorMatcher of undefined tns_modules/nativescript-angular/private_import_compiler.js line 9
```

I removed node_modules, platforms, and app from emulator. No luck

`"nativescript-angular": "^1.0.1",`

Any idea?

**Comment by matart15** [comment]
Ok it happens on "version": "1.1.0",

**Comment by NickIliev** [comment]
Hello @matart15 

We are working on this issue that occurs after new angular release.
Meanwhile as a workaround remove all tilde keys from your @angular dependencies in package.json file.
e.g.
```
    "@angular/common": "2.1.0",
    "@angular/compiler": "2.1.0",
    "@angular/core": "2.1.0",
    "@angular/forms": "2.1.0",
    "@angular/http": "2.1.0",
    "@angular/platform-browser": "2.1.0",
    "@angular/platform-browser-dynamic": "2.1.0",
    "@angular/router": "3.1.0",
```

**Comment by NickIliev** [comment]
The issue is now fixed with nativescript-angular 1.1.1.

    "@angular/common": "2.1.1",
    "@angular/compiler": "2.1.1",
    "@angular/core": "2.11",
    "@angular/forms": "2.1.1",
    "@angular/http": "2.1.1",
    "@angular/platform-browser": "2.1.1",
    "@angular/platform-browser-dynamic": "2.1.1",
    "@angular/router": "3.1.1",
    "nativescript-angular": "1.1.1",
